### PR TITLE
docs - api docs update

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Updates API docs for master branch.

### CODE REVIEW

File: docs/api/cloud-migration.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## `GET /api/cloud-migration/`
+
+Get the latest cloud migration, if any.
+
+You must be a superuser to do this.
+
+### PARAMS:
+
+-  **`_`**
```
Comment (alexyarosh) [inline docs/api/cloud-migration.md:25]: what does this mean?

File: docs/api/dashboard.md [hunk 2]
```diff
@@ -306,41 +322,9 @@ Update a Dashboard, and optionally the `dashcards` and `tabs` of a Dashboard. Th
...
+-  **`dash-updates`** map where {:name (optional) -> <nullable value must be a non-blank string.>, :description (optional) -> <nullable string>, :caveats (optional) -> <nullable string>, :points_of_interest (optional) -> <nullable string>, :show_in_getting_started (optional) -> <nullable boolean>, :enable_embedding (optional) -> <nullable boolean>, :embedding_params (optional) -> <nullable value must be a valid embedding params map.>, :parameters (optional) -> <nullable sequence of parameter must be a map with :id and :type keys>, :position (optional) -> <nullable value must be an integer greater than zero.>, :width (optional) -> <enum of fixed, full>, :archived (optional) -> <nullable boolean>, :collection_id (optional) -> <nullable value must be an integer greater than zero.>, :collection_position (optional) -> <nullable value must be an integer greater than zero.>, :cache_ttl (optional) -> <nullable value must be an integer greater than zero.>, :dashcards (optional) -> <nullable value must be seq of maps in which ids are unique>, :tabs (optional) -> <nullable value must be seq of maps in which ids are unique>}.
```
Comment (alexyarosh) [inline docs/api/dashboard.md:327]: format seems messed up

File: docs/api/ee/query-reference-validation.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+List of cards that have an invalid reference in their query. Shape of each card is standard, with the addition of an
+  `errors` key. Supports pagination (`offset` and `limit`), so it returns something in the shape:
+
+  ```
+    {:total  200
+     :data   [card1, card2, ...]
+     :limit  50
+     :offset 100
+  ```.
```
Comment (alexyarosh) [inline docs/api/ee/query-reference-validation.md:21]: The period after the code snippet is messing up markdown

**Review by alexyarosh — APPROVED** [review]
Approve to unblock but there are a couple of places with broken markdown

File: docs/api/cloud-migration.md (hunk 1 above)
Comment (jeff-bruemmer) [reply docs/api/cloud-migration.md]: Updated the script to handle this.

File: docs/api/dashboard.md (hunk 2 above)
Comment (jeff-bruemmer) [reply docs/api/dashboard.md:327]: We're relying on malli's `describe` function to produce the string here. We should tackle this in a subsequent PR.

File: docs/api/ee/query-reference-validation.md (hunk 3 above)
Comment (jeff-bruemmer) [reply docs/api/ee/query-reference-validation.md]: This should be handled now.

**Review by Playgirlkaybraz11 — COMMENTED** [review]
#47229 #47250 #47327 #39355 #5136 #47332 #9539 #30602 #20624

**Pull request merged**